Resources/public/mapbender.element.digitizer.js

```diff
@@ -224,54 +224,7 @@
      * @copyright 20.04.2015 by WhereGroup GmbH & Co. KG
      */
     $.widget("mapbender.mbDigitizer", {
-        options: {
```

Comment:
This `options` object is where the backend configuration for the element will land. Removing it seems a bit dangerous. Digitizer configuration is totally freeform, but it is still useful to have placeholders and defaults for the most common values in the code, for introspection and linter sanity at least.

I don't really like this removal.

Reply:
Well, the problem is, that the backend configuration of the digitizer consists only of `schemes`, i.e. attributes that don't interfere with  the hard-coded configuration in `options` - there is simply no intersection between backend configuration and the default values oft the `options`object. Therefore, leaving the `options` object as it is, is rather misleading. What I did to preserve default values for configuration attributes is placing them in the prototype for schemes, which I see as much more appropriate. I see it would be helpful to keep up with the given structure by provding at least an empy `options` object. What do you recommend, considering that?